--- CMakeLists.txt	2019-01-08 22:31:05.259303923 +0100
+++ CMakeLists.txt.new	2019-01-08 22:34:55.201310081 +0100
@@ -114,8 +114,11 @@
         IF(OPENMP_FOUND)
             set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS} -Werror=unknown-pragmas")
         ENDIF()
-        FIND_PACKAGE(Eigen3 REQUIRED)
-        INCLUDE_DIRECTORIES(${EIGEN3_INCLUDE_DIR})
+#        FIND_PACKAGE(Eigen3 REQUIRED)
+#        INCLUDE_DIRECTORIES(${EIGEN3_INCLUDE_DIR})
+	find_package( PkgConfig )
+	pkg_check_modules( EIGEN3 REQUIRED eigen3 )
+	include_directories( ${EIGEN3_INCLUDE_DIRS} )
     ELSE()
         SET(LIBRAW_INCLUDE_DIR "${CMAKE_SOURCE_DIR}/../DEPs/include/libraw")
         SET(FFTWF_INCLUDE_DIR "${CMAKE_SOURCE_DIR}/../DEPs/include/fftw3")
@@ -156,8 +159,11 @@
     FIND_PACKAGE(lcms2 REQUIRED)
     INCLUDE_DIRECTORIES(${LCMS2_INCLUDE_DIR})
 
-    FIND_PACKAGE(Eigen3 REQUIRED)
-    INCLUDE_DIRECTORIES(${EIGEN3_INCLUDE_DIR})
+#    FIND_PACKAGE(Eigen3 REQUIRED)
+#    INCLUDE_DIRECTORIES(${EIGEN3_INCLUDE_DIR})
+	find_package( PkgConfig )
+	pkg_check_modules( EIGEN3 REQUIRED eigen3 )
+	include_directories( ${EIGEN3_INCLUDE_DIRS} )
 ENDIF()
 
 IF(WIN32)
